package/setup.py

```diff
@@ -1,53 +1,47 @@
 # type: ignore
-import json
-import os
-from os import path
-
-from setuptools import find_packages, setup
-
-name = "kedro-viz"
-here = path.abspath(path.dirname(__file__))
-
-
-jsbuild = [
-    os.path.join(dirpath, f)
-    for dirpath, dirnames, files in os.walk("kedro_viz/html/")
-    for f in files
+from setuptools import setup
+
+extras_require["test"] = [
```

Comment:
You can have these under 

```toml
[project.optional-dependencies]
test = [
    ...
]
```

see https://github.com/kedro-org/kedro-plugins/blob/b946eec6a4e5979a3302e1a682c006a4411bddaa/kedro-airflow/pyproject.toml#L25-L40 for example